pkg/serializer/internal/metrics/sketch_benchmark_test.go

```diff
@@ -24,6 +25,7 @@ func benchmarkSplitPayloadsSketchesSplit(b *testing.B, numPoints int) {
 	for i := 0; i < numPoints; i++ {
 		testSketchSeries.Append(Makeseries(200))
 	}
+	c := logmock.New(b)
```

Comment:
Please use descriptive variable names, especially when scope spans multiple lines
```suggestion
	logger := logmock.New(b)
```